service/xmpp/XMPPEvents.js

```diff
@@ -120,6 +120,10 @@ var XMPPEvents = {
     /**
      * Indicates that the local sendrecv streams in local SDP are changed.
      */
-    SENDRECV_STREAMS_CHANGED: "xmpp.sendrecv_streams_changed"
+    SENDRECV_STREAMS_CHANGED: "xmpp.sendrecv_streams_changed",
+    /**
+     * Indicates that max users limit have been reached.
+     */
+    ROOM_MAX_USERS_ERROR: "xmpp.room_max_users_error"
```

Comment:
This commit adds only one event, but on a side note all of them should be either sorted in alphabetical order or grouped in some meaningful way, cause now they are just scattered around randomly